src/test/java/org/reaktivity/specification/nukleus/kafka/streams/MergedIT.java

```diff
@@ -436,9 +411,9 @@ public void shouldUnmergedFetchServerSentResetAndAbortWithMessage() throws Excep
 
     @Test
     @Specification({
-        "${scripts}/unmerged.fetch.filter.age.historical/client",
-        "${scripts}/unmerged.fetch.filter.age.historical/server"})
-    public void shouldFetchUnmergedMessagesWithAgeHistoricFilter() throws Exception
+        "${scripts}/merged.fetch.not.equals.filter.header/client",
+        "${scripts}/merged.fetch.not.equals.filter.header/server"})
+    public void shouldFetchMergedMessagesWithHeaderNotEqualsFilter() throws Exception
```

Comment:
This should follow the naming convention of `merged.fetch.filter.not.header`
Please add a scenario for `merged.fetch.filter.not.key` as well.